static/js/query-configs/websites.ts

```diff
@@ -36,9 +38,32 @@ export const getTransformedWebsiteName = (
   return transformedWebsiteKeys[0]
 }
 
+export interface WebsiteListingResponse {
+  count: number
+  next: string | null
+  previous: string | null
+  results: Website[]
+}
+type WebsitesListing = Record<string, WebsiteListingResponse>
+export const websiteListingRequest = (offset: number): QueryConfig => ({
+  url:       siteApiListingUrl(offset),
+  transform: (body: WebsiteListingResponse) => ({
+    websitesListing: {
+      [`${offset}`]: body
+    }
+  }),
+  update: {
+    websitesListing: (prev: WebsitesListing, next: WebsitesListing) => ({
```

Comment:
would it be worthwhile to put the website objects themselves into the `websites` entity for consistency? the pages could be stored here as arrays of website IDs or something along those lines. I think we did that somewhere in Open.

Reply:
Sure, I'll look into that

Reply:
Actually I'm not sure this is feasible since the website objects are slightly different between the listing and detail view (the `is_admin` field). If we populated the detail view with websites fetched in the list view, they would all implicitly have a falsy value for `is_admin`.

Reply:
won't the detail view make the request to that endpoint and fetch that detail record though?

Reply:
Right, I guess we can set `force: true` to make sure the new data is always up to date. I'll try that out

Reply:
We could also add `is_admin` to the list view and just fix the performance issues that we find. But I don't know if there are plans to add more data to the website serializer down the line